modules/schema/src/main/scala/org/tessellation/schema/transaction.scala

```diff
@@ -67,7 +67,7 @@ object transaction {
     val _Ordinal: Lens[TransactionReference, TransactionOrdinal] = GenLens[TransactionReference](_.ordinal)
 
     def of[F[_]: Async: KryoSerializer](signedTransaction: Signed[Transaction]): F[TransactionReference] =
-      signedTransaction.value.hashF.map(TransactionReference(signedTransaction.ordinal, _))
+      signedTransaction.value.hashF.map(a => TransactionReference(signedTransaction.ordinal, a))
```

Comment:
If `_` doesn't work anymore maybe call the value `h` or `hash` instead of `a`

Reply:
ah, good catch. it's a leftover

Reply:
fixed :+1: